src/main/logutils.ts

```diff
@@ -416,13 +519,25 @@ function handleEncounterStartLine (line: LogLine): void {
  * Handle a line from the WoW log. 
  */
 function handleEncounterStopLine (line: LogLine): void {
-    if (isChallengeModeActive) {
-        console.log("[Logutils] ENCOUNTER_END in an active Mythic Keystone dungeon is ignored.")
-        return;
-    }
-    
     const videoStopDate = line.date();
     const encounterResult = Boolean(parseInt(line.args[5], 10));
+    const encounterID = parseInt(line.args[1], 10);
+
+    if (recorder.isRecording && activeChallengeMode) {
+        const currentSegment = activeChallengeMode.getCurrentVideoSegment()
+        if (currentSegment) {
+            currentSegment.result = encounterResult
+        }
+
+        const vSegment = new ChallengeModeVideoSegment(
+            VideoSegmentType.Trash, videoStopDate, getRelativeTimestampForVideoSegment(videoStopDate)
+        )
+
+        // Add a trash segment as the boss encounter ended
+        activeChallengeMode.addVideoSegment(vSegment, videoStopDate);
```

Comment:
What about dungeons where bosses into boss is pulled. I'm almost certain this happens with some routes.

Reply:
I've done a lot of dungeons and I've never seen a boss being pulled _into_ a boss. Do you have an example?

Reply:
I had Taza'vesh in mind but I'm not sure as I basically never do M+.

 As per earlier happy enough if there is a few rough edges like this here.

Reply:
Especially Taza'vesh will have a skip before the first boss then, and although it's not a complicated skip by any stretch of imagination, I'd personally like to be able to review it if we screw it up :p 

The actual reason for having a first timeline segment before the first boss is just to simply be able to seek to 0:00 withuot having to use the video player seek bar.

Reply:
Ah so it would just show like "trash" 0-30s but there would be no combat in that, just running into position? Actually I like that as it is then. I agree that's important stuff you want to see and be able to skip to!